static/js/pages/legal.js

```diff
@@ -1,10 +1,15 @@
 'use strict';
 
 /* global require */
+require('jquery.inputmask');
+require('jquery.inputmask/dist/inputmask/inputmask.date.extensions.js');
+require('jquery.inputmask/dist/inputmask/inputmask.numeric.extensions.js');
```

Comment:
Responding to @vrajmohan 's question on my previous PR,`inputmask.numeric.` isn't required. Copy-and-paste mistake. :)